pyproject.toml

```diff
@@ -11,6 +11,8 @@ gunicorn = "^20.1.0"
 requests = "^2.28.1"
 Flask = "^2.1.2"
 pytest = "^7.1.2"
+pymongo = "^4.3.2"
+mongomock = "^4.1.2"
```

Comment:
We could move this into the `dev-dependencies` section below, so that it doesn't get installed in our production container; the effect will be fairly minor but it should reduce the size of our image _slightly_ and as importantly it removes a potential attack vector; both of which will keep scaling as we add more libraries so it's a good habit to watch out for

Edit: This is only really true if we adjust the Dockerfile slightly so that it install dependencies with the `--no-dev` [(or more recent equivalents)](https://python-poetry.org/docs/cli/#options-2) argument for our production container